decidim-core/app/controllers/decidim/authorization_modals_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  class AuthorizationModalsController < Decidim::ApplicationController
+    helper_method :status, :authorize_action_path
+    layout false
+
+    def show
+      render
```

Comment:
I don't think you need the `render` as long as there's an unlerlying template with the right name. I actually learnt from @oriolgual that you don't even new the action at all, as long as the template exists with the right name, although I find that style very obscure...